ansible/roles/operator-pipeline/templates/openshift/tasks/commit-pinned-digest.yml

```diff
@@ -3,21 +3,95 @@ apiVersion: tekton.dev/v1beta1
 kind: Task
 metadata:
   name: commit-pinned-digest
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.21.0"
+    tekton.dev/categories: Git
+    tekton.dev/tags: git
+    tekton.dev/displayName: "commit-pinned-digest"
+
 spec:
   workspaces:
     - name: source
+    - name: ssh-directory
+      optional: true
+      description: |
+        A .ssh directory with private key, known_hosts, config, etc. Copied to
+        the user's home before git commands are executed. Used to authenticate
+        with the git remote when performing the task to commit changes.
+        Binding a Secret to this Workspace is strongly recommended over other
+        volume types.
   params:
     - name: dirty_flag
+    - name: verbose
+      description: Log the commands that are executed during `commit-pinned-digest`'s operation.
+      type: string
+      default: "true"
+    - name: git_user_name
+      type: string
+      description: |
+        Git user name for performing git operation.
+      default: "digestPinning"
+    - name: git_user_email
+      type: string
+      description: |
+        Git user email for performing git operation.
+      default: "no-reply@redhat.com"
+  results:
+    - name: commit
+      description: The precise commit SHA that was fetched by this Task.
+    - name: branch
   steps:
     - name: commit-changes
-      image: registry.access.redhat.com/ubi8-minimal
+      image: docker.io/alpine/git:v2.26.2@sha256:23618034b0be9205d9cc0846eb711b12ba4c9b468efdd8a59aac1d7b1a23363f #tag: v2.26.2
```

Comment:
Any specific reason why to use an alpine image? And why there are both tags and digest?

Reply:
Good point. If it's possible to use ubi, we should.